sdk/program/src/vote/state/mod.rs

```diff
@@ -398,6 +431,40 @@ impl VoteState {
         self.epoch_credits.last_mut().unwrap().1 += credits;
     }
 
+    // Computes the vote latency value to use for a vote on a slot where the vote itself landed in the given slot
+    pub fn compute_vote_latency(voted_for_slot: Slot, voted_in_slot: Slot) -> u8 {
```

Comment:
nit: voted_in_slot -> current_slot

Reply:
OK I have incorporated your feedback.  Note that I also updated the solana cli to show the vote latencies of votes if they have been recorded.

Reply:
> nit: voted_in_slot -> current_slot

Actually on thinking about this more ... I think my previous version that kept a parallel structure was better, because the change that alters the Lockout structure means that the structure of a Vote account is now altered such that members that were at fixed offsets previously have moved to new offsets (because the 'votes' VecDeque will now be a different size than it was previously).

A change that adds a new member to the end of the VoteState structure doesn't alter the size of the VoteState account nor the placement of any pre-existing data members, it just adds something new to the end.  I think that's a much less disruptive change.


Can we talk about this?  I'd like to revert to my previous version of the change for this reason.